src/OpenTK.NT/Native/User32/Enums/WindowMessageActivate.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace OpenTK.NT.Native
+{
+    public enum WindowMessageActivate // typedef in windows?
```

Comment:
Is it or is it not? Remove comment if we've decided to use it as an enum, or document it.

Reply:
[It is](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646274(v=vs.85).aspx).

It seems to be unused (I checked on 4.0 and develop) so I'll remove it.

Reply:
Needs another look.